<launch>

<node pkg = "qidian_tactics" type = "tactics" name = "tactics"  output="screen">
        <!--x,y,z,w-->
        <param name="Task_0_point"  value="(5.02,2.41,1.00,0.03),(1.56,2.60,1,0.03)" />
        <param name="Task_1_point"  value="(3.49,-0.24,0.04,1.00)" />
        <param name="Task_2_point"  value="(6.49,0.64,-0.01,1.00)" />
        <param name="Task_3_point"  value="(1.36,-1.87,-0.05,1.00)" />
        <param name="Task_4_point"  value="(-1.84,2.20,-0.03,1)" />
        <param name="Task_5_point"  value="(6.49,0.64,-0.01,1.00)" />
</node>





<!-- [ INFO] [1710643418.941498635, 928.641000000]: 读取到新的导航记录 Point : 1.36 -1.87 -0.05 1.00
[ INFO] [1710643466.131237022, 975.674000000]: 读取到新的导航记录 Point : 3.49 -0.24 0.04 1.00
[ INFO] [1710643516.339172792, 1025.689000000]: 读取到新的导航记录 Point : 6.49 0.64 0.39 0.92
[ INFO] [1710643591.355442639, 1100.467000000]: 读取到新的导航记录 Point : 5.02 2.41 1.00 0.03
[ INFO] [1710643666.488192830, 1175.357000000]: 读取到新的导航记录 Point : 1.56 2.60 1.00 0.01
[ INFO] [1710644264.169776146, 1771.466000000]: 读取到新的导航记录 Point : -2.20 2.61 -0.03 1.00-->




</launch>